modelexpress_client/python/modelexpress/topology.py

```
@@ -0,0 +1,73 @@
+    Best-effort: an unset or unparseable value yields ``{}`` (this node then
+    shares no domain with any source, i.e. rendezvous ordering).
+    """
+    if raw is None:
+        from . import envs
+
+        raw = envs.MX_P2P_TOPOLOGY
+    if not raw:
+        return {}
+    try:
+        parsed = json.loads(raw)
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)